script.module.resolveurl/lib/resolveurl/lib/helpers.py

```diff
@@ -183,7 +183,7 @@ def __parse_to_list(_html, regex):
     return source_list
 
 
-def get_media_url(url, result_blacklist=None, patterns=None, generic_patterns=True, referer=True, redirect=True):
+def get_media_url(url, result_blacklist=None, patterns=None, generic_patterns=True, referer=True, redirect=True, verify=True):
```

Comment:
I think it is better to call this `verifypeer` instead of `verify` as it clashes with the other place where `verify` could be used in NET
or if you use `verify` use it for the NET init too with no verify so that one parameter does both disabling ssl cert check to the embed site as well as to the final hoster site.

Reply:
Done :)